incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/airline/ConfigGenerator.java

```diff
@@ -58,7 +60,18 @@ public abstract class ConfigGenerator
 
     public abstract String generate();
 
-    protected String resolveContentType()
+    protected boolean jsonContentType()
+    {
+        boolean result = false;
+        String contentType = resolveContentType();
```

Comment:
If `resolveContentType` not used elsewhere, then let's inline here instead.

Reply:
The `resolveContentType` method is specific to AsyncAPI. I will have two different implementations very soon (in the next PR) for OpenApi vs AsyncAPI, while the `hasJsonContentType` method will stay the same, so I'd prefer not to inline.